frameworks.md

```diff
@@ -81,11 +86,9 @@ Modules particularities:
   * the start and end of a module, even the global one, are emitted only if the suite itself contains at least one test
   * nested modules have a concatenated name, from the outer most suite to the inner most
 
-Nested modules execution is done the following way:
-  * take in order the modules inside the top level module
-  * than execute them from the inner most module to the outer most
+The execution is done in the source order, but QUnit has a more flat style for nested modules, it emit the start of a module, emit its tests, then the module ends and starts another, even if the modules were nested, there is not a sort of recursion between the modules.
 
-Respecting the rules above the tests in the top level suite will be the lasts that will execute.
+**Although the above execution, the QUnit default reporter is always displaying only the suites in the source order**, tests are displayed together with their parent module which breaks the source order, this applies also for random tests execution (check out below example).
```

Comment:
I think "Despite" would work better here. Like "Despite the execution order, ...